handlers/read.go

```diff
@@ -14,10 +14,19 @@ func MakeReadHandler(client *containerd.Client, serviceMap *ServiceMap) func(w h
 
 		res := []types.FunctionStatus{}
 		for _, k := range serviceMap.Keys() {
+			i, _ := serviceMap.containers[k].Image(serviceMap.context)
+			t, _ := serviceMap.containers[k].Task(serviceMap.context, nil)
+			svc, _ := t.Status(serviceMap.context)
+			replicas := 0
+			if svc.Status == "running" {
+				replicas = 1
+			}
+
 			res = append(res, types.FunctionStatus{
-				Name:     k,
-				Replicas: 1,
-				// Namespace: "openfaas-fn",
+				Name:      k,
+				Image:     i.Name(),
+				Replicas:  uint64(replicas),
+				Namespace: "openfaas-fn",
```

Comment:
eventually I want to extract this variable so that it can be changed.

Reply:
I'll refactor all this and put everything that's network related in the same file.

Reply:
No need to do that now. Best control the amount of changes.